forc-fmt/src/main.rs

```diff
@@ -28,6 +28,9 @@ pub struct App {
     /// Path to the project, if not specified, current working directory will be used.
     #[clap(short, long)]
     pub path: Option<String>,
+    /// Maximum length of line
+    #[clap(short = 'l', long = "length")]
+    pub line_length: Option<u32>,
```

Comment:
Rather than adding an option that will be removed later, I would hard-code the length to a constant `80` or `100` for now. Then later we can make it a configurable parameter in `swayfmt.toml` #605.

Reply:
Checkout my recent commit! #1425 nearly have the `swayfmt.toml` ready I believe. If possible, anything relating to the current formatter we may want to hold off on. The foundation is almost ready, and we can start building there.